### PR TITLE
Geoip updates

### PR DESCRIPTION
**Pull request by jencub**
Updates for GeoIP and Cloudflare.

https://jira.greenpeace.org/browse/PLANET-4606 
https://jira.greenpeace.org/browse/PLANET-4643

### CODE REVIEW

File: src/planet-4-151612/openresty/bin/nginx_configure_cloudflare.sh [hunk 1]
```diff
@@ -27,9 +27,8 @@ set -euo pipefail
 
 _good "$(printf "%-10s " "openresty:")" "CLOUDFLARE ${CLOUDFLARE_ENABLED}"
 
-[[ ${CLOUDFLARE_ENABLED} = "true" ]] || {
```
Comment (27Bslash6) [inline src/planet-4-151612/openresty/bin/nginx_configure_cloudflare.sh]: Please keep original branching / exit logic

File: src/planet-4-151612/openresty/bin/nginx_configure_geoip2.sh [hunk 2]
```diff
@@ -1,29 +1,58 @@
 #!/usr/bin/env bash
 set -euo pipefail
+#Check License and Account key variables
 
-_good "$(printf "%-10s " "openresty:")" "GEOIP2 ${GEOIP2_ENABLED}"
-
-[[ ${GEOIP2_ENABLED} = "true" ]] || {
+if [[ ${PHP_ENABLED} = 'true' ]]
```
Comment (27Bslash6) [inline src/planet-4-151612/openresty/bin/nginx_configure_geoip2.sh]: Please simplify exit logic, keep conditionals at single depth.

Test for GEOIP_ENABLED || exit quietly
Test for PHP_ENABLED && exit with failure
Test License data || exit failure

... proceed with script

File: src/planet-4-151612/openresty/templates/Dockerfile.in [hunk 3]
```diff
@@ -15,6 +15,8 @@ ENV \
     CLOUDFLARE_ENABLED="false" \
     GEOIP2_ENABLED="false" \
     GEOIP2_VERSION="3.2" \
+    MAXMIND_ACCOUNTID="${MAXMIND_ACCOUNTID}" \
```
Comment (27Bslash6) [inline src/planet-4-151612/openresty/templates/Dockerfile.in]: Remove template variables, leave as `=""`

File: src/planet-4-151612/openresty/templates/README.md.in [hunk 4]
```diff
@@ -43,6 +43,11 @@ APP_GROUP                  | nginx   | Service group name
 UPLOAD_MAX_SIZE            | ${UPLOAD_MAX_SIZE}     | Sets `nginx_client_max_body_size`
 OPENRESTY_MAX_WORKER_PROCESSES | ${OPENRESTY_MAX_WORKER_PROCESSES}       | Sets `worker_processes`, will not exceed number of logical cores
 CHOWN_APP_DIR              | false   | If true `chown` `${PUBLIC_PATH}` as `APP_USER:APP_GROUP`
+MAXMIND_ACCOUNTID          | \_\_DISABLED\_\_   | Maxmind account, can be regenerated via [Maxmind.com](https://www.maxmind.com/en/accounts/current/people/current/)
```
Comment (27Bslash6) [inline src/planet-4-151612/openresty/templates/README.md.in]: beep

File: tests/src/planet-4-151612/handbook/tests/env.bash [hunk 5]
```diff
@@ -18,7 +18,7 @@ export image
 APP_HOSTNAME="www.planet4.test"
 DB_IMAGE="mysql:5.7"
 GIT_SOURCE="https://github.com/greenpeace/planet4-base-fork"
-GIT_BRANCH="develop"
+GIT_BRANCH="build-test"
```
Comment (27Bslash6) [inline tests/src/planet-4-151612/handbook/tests/env.bash]: Not required anymore

File: tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf [hunk 6]
```diff
@@ -5,10 +5,10 @@
 # Account is currently registered with jhasenau@greenpeace.org
 
 # `AccountID` is from your MaxMind account.
-AccountID 146520
```
Comment (27Bslash6) [inline tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf]: File not required

File: tests/src/planet-4-151612/wordpress/tests/env.bash [hunk 7]
```diff
@@ -18,7 +18,7 @@ export image
 APP_HOSTNAME="www.planet4.test"
 DB_IMAGE="mysql:5.7"
 GIT_SOURCE="https://github.com/greenpeace/planet4-base-fork"
-GIT_BRANCH="develop"
+GIT_BRANCH="build-test"
```
Comment (27Bslash6) [inline tests/src/planet-4-151612/wordpress/tests/env.bash]: Not required

File: tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf (hunk 6 above)
Comment (jencub) [reply tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf]: I moved this file into tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf where it is needed now.

**Pull request merged**